Typos `kubesimplfy's` fixed in README.md (#159)
kubesimplify/website: README.md

<h1>Contributing Guidelines: </h1>

We're a warm and welcoming community of open source contributors. Please join. All types of contributions are welcome. Be sure to read the Kubesimplify <a href="https://github.com/kubesimplify/website/blob/main/CONTRIBUTION.md">Contribution.md</a> for a tour of resources available to you and how to get started.

> Kubesimplify's new website. This codebase is what runs on [Kubesimplify](https://kubesimplify.github.io/website/). 

Reviewer: ```suggestion
This is the source code of the official [Kubesimplify's website](https://kubesimplify.github.io/website/). 
```
